### PR TITLE
`Forms`: GroupElement tests

### PR DESCRIPTION
**Pull request by kaushikrw**
### Summary of changes

Introduces `GroupFormElement` UI tests based on the common test design.

### CODE REVIEW

File: toolkit/featureforms/src/androidTest/java/com/arcgismaps/toolkit/featureforms/GroupElementTests.kt [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+     * https://devtopia.esri.com/runtime/common-toolkit/blob/main/designs/Forms/FormsTestDesign.md#test-case-61-test-initially-expanded-and-collapsed
+     */
+    @Test
+    fun testInitialState() {
+        val groupFormElement1 = getGroupElementWithLabel("Group with Multiple Form Elements")
+        val groupElement1 = composeTestRule.onNodeWithText(groupFormElement1.label)
+        groupElement1.assertIsDisplayed()
+        // assert description is displayed
+        groupElement1.assertTextContains(groupFormElement1.description)
+        //groupElement1.assertContentDescriptionContains("expanded")
```
Comment (sorenoid) [inline toolkit/featureforms/src/androidTest/java/com/arcgismaps/toolkit/featureforms/GroupElementTests.kt:86]:
```suggestion

```

File: toolkit/featureforms/src/androidTest/java/com/arcgismaps/toolkit/featureforms/GroupElementTests.kt [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+        assert(groupElement1.isToggled())
+        // assert this group has children including the header
+        assert(groupElement1.onParent().onChildren().fetchSemanticsNodes().count() > 1)
+
+        val groupElement2 = composeTestRule.onNodeWithText("Group with Multiple Form Elements 2")
+        groupElement2.assertIsDisplayed()
+        //groupElement2.assertContentDescriptionContains("collapsed")
```
Comment (sorenoid) [inline toolkit/featureforms/src/androidTest/java/com/arcgismaps/toolkit/featureforms/GroupElementTests.kt:93]:
```suggestion

```

**Review by sorenoid — APPROVED** [review]
very nice. minor cleanup but LGTM.

**Comment by kaushikrw** [comment]
Thanks @sorenoid. Merging!

**Pull request merged**